src/hypothesis/extra/django/models.py

```diff
@@ -123,8 +123,11 @@ def _get_strategy_for_field(f):
                   'ipv4': ip4_addr_strings(), 'ipv6': ip6_addr_strings()}
         strategy = lookup[f.protocol.lower()]
     elif type(f) in (dm.TextField, dm.CharField):
-        strategy = st.text(min_size=(None if f.blank else 1),
-                           max_size=f.max_length)
+        strategy = st.text(
+            alphabet=st.characters(blacklist_characters=u'\x00'),
```

Comment:
Nit: It doesn't make much difference now, but maybe more naturally done as `min_codepoint=1`?

Reply:
Conceptually I think of this as a particular character that has odd effects, rather than being an invalid part of the range of codepoints.  If anyone has *strong* feelings about this I'm happy to change it, but otherwise IMO fine as-is.

Reply:
Fine by me.